webapp/channels/src/components/common/hooks/useOpenSalesLink.ts

```diff
@@ -31,8 +33,9 @@ export default function useOpenSalesLink(): [() => void, string] {
     }
 
     const contactSalesLink = buildMMURL(LicenseLinks.CONTACT_SALES, firstName, lastName, companyName, customerEmail, utmSource, utmMedium);
-    const goToSalesLinkFunc = () => {
+    const goToSalesLinkFunc = useCallback(() => {
         goToMattermostContactSalesForm(firstName, lastName, companyName, customerEmail, utmSource, utmMedium);
-    };
+    }, [companyName, customerEmail, firstName, lastName, utmMedium]);
```

Comment:
```suggestion
    }, [firstName, lastName, companyName, customerEmail, utmMedium]);
```

Just makes it easier to read when comparing with the function arguments